docs/requirements-rtd.txt

```diff
@@ -0,0 +1,4 @@
+sphinx==5.0.0
+sphinx_rtd_theme==1.0.0
+readthedocs-sphinx-search==0.1.1
```

Comment:
Version for `readthedocs-sphinx-search` should be bumped to at least 0.3.2 to resolve the security issue